### PR TITLE
docs - update info collection page

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
This PR simplifies the (out-of-date) information collection page.

The original page was too long, and keeping the page up to date is likely unnecessary (given that our source is open), and difficult in practice.

I removed the outdated table of events, and focused on the spirit of what this page was trying to communicate:

- We only collect Metabase data if you opt in.
- We don't collect your data.
- If you do give us data, thank you.
- Here's how to opt out.
- Here's a link to more info re: privacy and security.

### CODE REVIEW

File: docs/installation-and-operation/information-collection.md [hunk 1]
```diff
@@ -1,104 +1,29 @@
...
+title: About the anonymous usage data we collect
 redirect_from:
   - /docs/latest/information-collection
 ---
 
-# About the information we collect
+# About the anonymous usage data we collect
 
-Metabase uses Google Analytics and Snowplow to collect anonymous usage information from the installed servers that enable this feature. Below is a representative list of the events we have instrumented, as well as the information we collect about the user performing the action and the instance being used.
+If you've opted in to provide us with anonymous usage data (thank you!), Metabase will phone home some data collected via Google Analytics and Snowplow.
```
Comment (brunobergher) [inline docs/installation-and-operation/information-collection.md:9]: Pretty sure we don't use GA anymore ([asked here](https://metaboat.slack.com/archives/C05MPF0TM3L/p1710768894438609))

File: docs/installation-and-operation/information-collection.md [hunk 2]
```diff
@@ -1,104 +1,29 @@
...
 
-We also will phone home some anonymized metrics from the metabase application server on a nightly basis. Note that we don't collect any usernames, any emails, the server IP, database details of any kind or any personally identifiable information in this process.
+**We only collect Metabase data; we don't collect any of your data.** We don't collect any usernames, emails, server IPs, database details of any kind, or any personally identifiable information.
```
Comment (brunobergher) [inline docs/installation-and-operation/information-collection.md:11]: Worth linking to /security?

File: docs/installation-and-operation/information-collection.md (hunk 1 above)
Comment (brunobergher) [reply docs/installation-and-operation/information-collection.md:9]: We do, :/

File: docs/installation-and-operation/information-collection.md [hunk 3]
```diff
@@ -1,104 +1,29 @@
...
+The data helps us understand how people are actually using Metabase, which in turn helps us prioritize what to work on next.
 
-We collect this information to improve your experience and the quality of Metabase. We use the information you voluntarily share to understand how our users are actually using our product, what kinds of features to prioritize, and how many items to target in our design process. For example, without knowing what the distribution of the number of accounts on each instance in our install base is, we can't know whether the tools we provide can't scale until someone complains. And even then, we only hear complaints and not the people who are quietly happily using us. We're striving to create the best product possible.
+## Opting out of anonymous usage data collection
 
-If you prefer not to provide us with this anonymous usage data, you can always go to your instance’s admin section and toggle off the option for `Anonymous Tracking`.
+To opt out of providing us with your anonymous usage data:
```
Comment (brunobergher) [inline docs/installation-and-operation/information-collection.md:17]: We need to clarify that this is for self-hosted instances, [we're removing that opt in from Cloud](https://www.notion.so/metabase/Reduce-the-number-of-choices-in-onboarding-flow-f74849658dce4cceb9a9fd33b243f820?pvs=4#70f76b4db3a4410d920cfef1bb404236).

File: docs/installation-and-operation/start.md [hunk 4]
```diff
@@ -42,6 +42,10 @@ Export Metabase metrics for viewing with Prometheus.
 
 Rest in peace, Internet Explorer.
 
+## [About the anonymous usage data we collect](./information-collection.md).
+
+We collect Metabase data to improve the product. We don't collect your data.
```
Comment (brunobergher) [inline docs/installation-and-operation/start.md:47]: I think we can be more emphatic. Maybe
> We collect anonymous Metabase usage data to improve the product. We don't collect your data.

File: docs/installation-and-operation/information-collection.md [hunk 5]
```diff
@@ -1,104 +1,29 @@
...
+1. Click on the gear icon.
+2. Select **Admin settings**.
+3. Go to the **Settings** tab.
+4. Click **General**
+5. Toggle the **Anonymous tracking** option.
 
-In the list below, we spell out exactly why we collect each bit of information.
+If you're in the process of setting up your Metabase, you can also toggle off tracking during the `Usage Data Preferences` onboarding step. We collect a few anonymous events before that point, but won't do so anymore if you choose to opt out.
 
-## Example questions we want to answer
```
Comment (brunobergher) [inline docs/installation-and-operation/information-collection.md:23]: I think _something_ which explains why we collect and how we use this data seems important. Maybe something which covers:
- We collect events when relevant things happen in the product
- We collect some performance
- We phone home general instance stats like the number and types of DBs your have connected, the number of questions and dashboards in the instance, etc

Using those we're able to answer questions like
- What features are being used vs not
- Where users are getting stuck
- How is performance for key flows (like querying or loading) evolving over time

In a nutshell: assess opportunities for improvement and evaluate the outcome of our projects. It's not for sales or advertising (and it's anonymous, so we couldn't use it for that anyway ;) )

**Review by brunobergher — COMMENTED** [review]
Thanks for this first pass. Added comments on some stuff to work on still.

File: docs/installation-and-operation/information-collection.md [hunk 6]
```diff
@@ -1,104 +1,48 @@
...
+
+1. Click on the gear icon.
+2. Select **Admin settings**.
+3. Go to the **Settings** tab.
+4. Click **General**
+5. Toggle the **Anonymous tracking** option.
+
+If you're in the process of setting up your Metabase, you can also toggle off tracking during the `Usage Data Preferences` onboarding step. We collect a few anonymous events before that point, but won't do so anymore if you choose to opt out.
+
+If you're on Metabase Cloud, you cannot opt out of anonymous usage data collection entirely, as we require some standard SaaS usage data to run the service.
```
[the remark this answers is not in the document]
Comment (brunobergher) [reply docs/installation-and-operation/information-collection.md:44]: I think that phrasing draws fire. Can I propose that:
- We keep you original phrasing  for now
- You set. up a reminder to change it when the new setup flow ships

File: docs/installation-and-operation/start.md (hunk 4 above)
Comment (brunobergher) [reply docs/installation-and-operation/start.md:47]: ping (take it or leave, just making sure you saw it)

**Review by brunobergher — APPROVED** [review]
